crates/testenv/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "bdk_testenv"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+# For no-std, remember to enable the bitcoin/no-std feature
+# bitcoin = { version = "0.30", default-features = false }
```

Comment:
Does this commented line need to be here? I don't see how it is related to the above comment.